docs/languages/index.rst

```diff
@@ -0,0 +1,16 @@
+.. include:: /stub-topic.txt
+
+Programming languages on the .NET Platform
+------------------------------------------
+
+.. toctree::
+	:titlesonly:
+	
+    overview
+    c-sharp
+    f-sharp
+    visual-basic    
+	
```

Comment:
@blackdwarf @richlander @bleroy I think it would be good to add "More .NET languages" here and include at least a mention of PowerShell and other .NET languages from the community. At least a link to https://en.wikipedia.org/wiki/List_of_CLI_languages